### DISCUSSION TITLE
Problems to download from Google Classroom

### DISCUSSION THREAD

**Discussion by guutardochi** [aclap-dev/video-downloadhelper#936] [Help and Q&A]
### Your question

Hello goodnight! I'm having trouble downloading videos from Google Classroom. On the home page, there are several video links to click. In the first videos, when I click, a new tab opens, and VDH recognizes these videos, and I can download them. However, after a certain video, VDH no longer recognizes any. What should I do? Thanks!

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"4f25ffc28e724203"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "D:\\Medicina\\Formação\\Cursos\\2024 - Climatério e menopausa\\Encontro 16",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": true,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 3,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Are these purchased classes?  If not, it might be worth it to get a little scientific about this.  Try to observe exactly how many downloads work before it stops working.  Is it always the same number?  If you close & reopen all your Chrome windows, are downloads once again possible?  Is there a warning somewhere in the fine print that says there is a quota on how many classes you can download in a day?

If you can describe things in this kind of detail, I might be able to try this with Firefox.  (I don't use Chrome.)  But you'll have to be able to reliably repeat the problem.  And you'll have to give me the URLs of the classes that worked as well as the first one that didn't.  I'm assuming Google Classes is something anybody with a Google ID can access.  I have a GMail address so I'm guessing I would be able to test this.

**Reply by GrampaWildWilly** [reply]
I had never heard of Google Classroom so I checked it out.  It looks like it is a paid service over & above what I can look at with my generic free Google ID.  Oh well.  You're pretty much on your own here.  Too bad.
